.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "Java",
+
+	"image": "mcr.microsoft.com/devcontainers/java:0-17",
+
+	"features": {
+		"ghcr.io/devcontainers/features/java:1": {
+			"version": "none",
+			"installMaven": "true",
+			"installGradle": "false"
+		},
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {}
+	},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
```

Comment:
Shouldn't we uncomment this and forward 8080?

Reply:
In my opinion - there is no need because the port couldn't be used once the container is created , only when we build docker image and run it with the docker compose we can access the application using this port
Should I remove the comment ?

Reply:
Isn't this the same as port forwarding in compose? Am I getting something wrong here?

Reply:
The port forward specified in this file will happen soon as the devcontainer will start running. but as long we didn't run the docker compose - it is useless. 
the port forward for the app will be created automatically (only)  when we are run the docker compose. 
it can be add without causing any problem- but it might be confusing

Reply:
ok let's keep it as is